gratipay/models/team/mixins/takes.py

```diff
@@ -71,19 +72,23 @@ def set_take_for(self, participant, take, recorder, cursor=None):
 
         """
         def vet(p):
-            assert not p.is_suspicious, p.id
-            assert p.is_claimed, p.id
-            assert p.email_address, p.id
-            assert p.has_verified_identity, p.id
+            if p.is_suspicious:
+                raise NotAllowed(_("user must not be flagged as suspicious"))
```

Comment:
Hmm ... I was thinking that the UI should prevent users from ever hitting these messages in the first place, as you [suggest](https://github.com/gratipay/gratipay.com/pull/4077#issuecomment-236036025) with precluding non-verified, etc. users from `lookup.json`.